requirements.txt

```diff
@@ -5,3 +5,5 @@ django-apptemplates>=1.4,<2.0
 django-celery-email>=3.0.0,<3.1
 djangorestframework>=3.11,<3.12
 swapper
+#TODO : Remove after release of openwisp-notifications
+https://github.com/openwisp/openwisp-notifications/tarball/master
```

Comment:
as far as I know this doesn't work with setuptools, usually here we put just the version `openwisp-notification~=0.1.0`, and we install from github in the .travis.yml config